azavea/raster-foundry
# Fix old footprints which were placed in wrong column

## Overview

Old footprints were placed in the tile_footprint column instead of the data_footprint column where they belong

## Testing Instructions

 * Run Migrations
 * Open RF frontend and go to the browse view - it should now populate the scene list
